prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -827,18 +827,19 @@ ssize_t rxr_pkt_proc_matched_medium_rtm(struct rxr_ep *ep,
 					struct rxr_rx_entry *rx_entry,
 					struct rxr_pkt_entry *pkt_entry)
 {
-	struct rxr_pkt_entry *cur;
+	struct rxr_pkt_entry *cur, *nxt;
 	struct efa_mr *desc;
 	char *data;
 	size_t offset, hdr_size, data_size;
 
 	cur = pkt_entry;
 	while (cur) {
-		hdr_size = rxr_pkt_req_hdr_size(pkt_entry);
```

Comment:
Is this a separate bug? Is it possible for the header sizes to be different across packets in a medium message?

Reply:
Yes, this is a bug. will split it out a bug fix commit.